packages/force_update/mobile/lib/src/app_version.dart

```diff
@@ -0,0 +1,42 @@
+extension Compare<T> on Comparable<T> {
+  bool operator <=(T other) => compareTo(other) <= 0;
+  bool operator >=(T other) => compareTo(other) >= 0;
+  bool operator <(T other) => compareTo(other) < 0;
+  bool operator >(T other) => compareTo(other) > 0;
+}
+
+class AppVersion implements Comparable<AppVersion> {
+  const AppVersion({required this.version});
+
+  final String version;
+
+  @override
+  int compareTo(other) {
```

Comment:
Using this package https://pub.dev/packages/version seems safer than writing this logic from scratch. I believe there are quite a lot of edge-cases here

Reply:
replaced with the package